src/platforms/stm32/src/lib/gpio_driver.c

```diff
@@ -543,7 +556,7 @@ static Context *gpio_driver_create_port(GlobalContext *global, term opts)
     if (UNLIKELY(!globalcontext_register_process(ctx->global, atom_index, ctx->process_id))) {
         scheduler_terminate(ctx);
         AVM_LOGE(TAG, "Only a single GPIO driver can be opened.");
-        return create_pair(ctx, ERROR_ATOM, USED_ATOM);
+        return create_error_tuple(ctx, USED_ATOM);
```

Comment:
As in your other PR, this should return NULL.

Reply:
Fixed.